pkg/controller/monitor/monitor_controller.go

```diff
@@ -332,6 +336,20 @@ func (r *ReconcileMonitor) Reconcile(ctx context.Context, request reconcile.Requ
 		return reconcile.Result{}, err
 	}
 
+	// Query the KubeControllersConfiguration object. We'll use this to help configure kube-controllers metric port.
+	kubeControllersConfig := &crdv1.KubeControllersConfiguration{}
+	err = r.client.Get(ctx, types.NamespacedName{Name: "default"}, kubeControllersConfig)
+	if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
Could you extract this method into the utils package, so both core_controller and monitor_controller can use it?

Reply:
Done